auth-web/src/components/auth/staff/PPRLauncher.vue

```diff
@@ -17,20 +17,53 @@
   </v-card>
 </template>
 <script lang="ts">
-import { Component } from 'vue-property-decorator'
+import { Component, Watch } from 'vue-property-decorator'
 import ConfigHelper from '@/util/config-helper'
+import { KCUserProfile } from 'sbc-common-components/src/models/KCUserProfile'
 import Vue from 'vue'
+import { namespace } from 'vuex-class'
+const userModule = namespace('user')
 
 // FUTURE: import this from shared components once built
 // - this is converted statically from UserProduct.vue in bcgov/bcregistry repo
 @Component({})
 export default class PPRLauncher extends Vue {
-  private text = 'Register or search for legal claims on personal property.'
-  private title = 'Staff Personal Property Registry'
+  @userModule.State('currentUser') public currentUser!: KCUserProfile
+
+  private title = ''
+  private text = ''
+  private img = ''
 
   private get pprUrl (): string {
     return ConfigHelper.getPPRWebUrl()
   }
+
+  private getImgUrl (img) {
+    let images = require.context('@/assets/img/')
```

Comment:
let images is not re-assigning so can use const

Reply:
Good catch, thanks!

Updated